tests/lac/linear_operator_08.with_umfpack=off.with_lapack=true.output

```diff
@@ -47,24 +47,24 @@ DEAL:Solvers:Linear operator:cg::Starting value 16.88194302
 DEAL:Solvers:Linear operator:cg::Convergence step 1 value 0.000000000
 DEAL:Solvers::SolverFGMRES
 DEAL:Solvers:Standard solver:FGMRES::Starting value 16.88194302
-DEAL:Solvers:Standard solver:FGMRES::Convergence step 1 value 2.744725072e-15
+DEAL:Solvers:Standard solver:FGMRES::Convergence step 1 value 3.696939525e-15
 DEAL:Solvers:Linear operator:FGMRES::Starting value 16.88194302
-DEAL:Solvers:Linear operator:FGMRES::Convergence step 1 value 2.744725072e-15
+DEAL:Solvers:Linear operator:FGMRES::Convergence step 1 value 3.696939525e-15
 DEAL:Solvers::SolverGMRES
 DEAL:Solvers:Standard solver:GMRES::Starting value 8.440971508
-DEAL:Solvers:Standard solver:GMRES::Convergence step 1 value 1.372362536e-15
+DEAL:Solvers:Standard solver:GMRES::Convergence step 1 value 1.848469762e-15
 DEAL:Solvers:Linear operator:GMRES::Starting value 8.440971508
-DEAL:Solvers:Linear operator:GMRES::Convergence step 1 value 1.372362536e-15
+DEAL:Solvers:Linear operator:GMRES::Convergence step 1 value 1.848469762e-15
 DEAL:Solvers::SolverMinRes
 DEAL:Solvers:Standard solver:minres::Starting value 11.93733639
 DEAL:Solvers:Standard solver:minres::Convergence step 1 value 2.059168646e-15
 DEAL:Solvers:Linear operator:minres::Starting value 11.93733639
 DEAL:Solvers:Linear operator:minres::Convergence step 1 value 2.059168646e-15
 DEAL:Solvers::SolverQMRS
-DEAL:Solvers:Standard solver:QMRS::Starting value 16.88194302
-DEAL:Solvers:Standard solver:QMRS::Convergence step 1 value 0.000000000
-DEAL:Solvers:Linear operator:QMRS::Starting value 16.88194302
-DEAL:Solvers:Linear operator:QMRS::Convergence step 1 value 0.000000000
+DEAL:Solvers:Standard solver:SQMR::Starting value 16.88194302
```

Comment:
I thought I had a dyslexic moment when writing this test, but I note that the solver is indeed called `SolverQMRS`. So what's with this change (and the similar changes to the other tests)?

Reply:
Oh, this is just a naming issue. The original creators of the algorithm (Roland Freund and Noel Nachtigal) used the short form SQMR for this method ("QMR for symmetric matrices") and QMRS for another, different method ("QMR squared", named similarly to CGS). I think it's better to use the conventional name to avoid possible confusion. I at least changed the deallog name, thus these differences.